### ISSUE TITLE
Better Options

### ISSUE THREAD

**Issue by quick007** [SkriptLang/Skript#4032]
would be cool if we could do like
```
options:
    price:
        apple: 1000
        orange: 100
    among:
        us: lSkyblock
        sus: imposterrrr
```

you would reference options like `{@price.apple}`. This would make it easier for scripts with lots of configurations to work.

**Comment by officialMozart** [comment]
You can already do this with list variables

**Comment by quick007** [comment]
options !== variables

Most people use options for easy config or whatever, and variables in the actual code.

**Comment by TheLimeGlass** [comment]
Not completed

**Comment by quick007** [comment]
uh ok

**Comment by TheLimeGlass** [comment]
it actually was

**Comment by GStudiosX2** [comment]
lmfao

**Comment by Fusezion** [comment]
> it actually was

Sorry mistake on my end, changelog only mentions this in structure api not apart of the the other PR



> lmfao LimeGlass you good ? btw when was this implemented 2.7 ? 
Yes in the recently released 2.7-beta1 release
